### PR TITLE
Catchpoint alignment and github release action added

### PR DESCRIPTION
**Pull request by erdogan-yesil**

### CODE REVIEW

File: thundra/constants.py [hunk 1]
```diff
@@ -15,15 +15,15 @@
...
-TRIGGER_OPERATION_NAME_TAG = 'x-thundra-trigger-operation-name'
-TRIGGER_CLASS_NAME_TAG = 'x-thundra-trigger-class-name'
-TRIGGER_DOMAIN_NAME_TAG = 'x-thundra-trigger-domain-name'
-TRIGGER_RESOURCE_NAME_TAG = 'x-thundra-resource-name'
-
-THUNDRA_TRACE_ID_KEY = 'x-thundra-trace-id'
-THUNDRA_TRANSACTION_ID_KEY = 'x-thundra-transaction-id'
-THUNDRA_SPAN_ID_KEY = 'x-thundra-span-id'
-THUNDRA_BAGGAGE_PREFIX = 'x-thundra-baggage-'
+TRIGGER_OPERATION_NAME_TAG = 'x-catchpoint-trigger-operation-name'
```
Comment (serkan-ozal) [inline thundra/constants.py:18]: This is not the tag, but key. So it should be `TRIGGER_OPERATION_NAME_KEY`

File: thundra/constants.py [hunk 2]
```diff
@@ -15,15 +15,15 @@
...
+TRIGGER_CLASS_NAME_TAG = 'x-catchpoint-trigger-class-name'
```
Comment (serkan-ozal) [inline thundra/constants.py:19]: This is not the tag, but key. So it should be `TRIGGER_CLASS_NAME_KEY`

File: thundra/constants.py [hunk 3]
```diff
@@ -15,15 +15,15 @@
...
+TRIGGER_DOMAIN_NAME_TAG = 'x-catchpoint-trigger-domain-name'
```
Comment (serkan-ozal) [inline thundra/constants.py:20]: This is not the tag, but key. So it should be `TRIGGER_DOMAIN_NAME_KEY`

File: thundra/constants.py [hunk 4]
```diff
@@ -15,15 +15,15 @@
...
+TRIGGER_RESOURCE_NAME_TAG = 'x-catchpoint-resource-name'
```
Comment (serkan-ozal) [inline thundra/constants.py:21]: This is not the tag, but key. So it should be `TRIGGER_RESOURCE_NAME_KEY`

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install build
+    - name: Build package
+      run: python -m build
+    - name: Publish package to Test PyPI
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        password: ${{ secrets.TEST_PYPI_API_TOKEN }}
```
Comment (serkan-ozal) [inline .github/workflows/release.yml:35]: Update to `secrets.THUNDRA_PYPI_API_TOKEN`

File: .github/workflows/release.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        repository-url: https://test.pypi.org/legacy/
```
Comment (serkan-ozal) [inline .github/workflows/release.yml:36]: You can remove this input

File: thundra/constants.py (hunk 1 above)
Comment (erdogan-yesil) [reply thundra/constants.py]: done

File: thundra/constants.py (hunk 2 above)
Comment (erdogan-yesil) [reply thundra/constants.py]: done

File: thundra/constants.py (hunk 3 above)
Comment (erdogan-yesil) [reply thundra/constants.py]: done

File: thundra/constants.py (hunk 4 above)
Comment (erdogan-yesil) [reply thundra/constants.py]: done

**Review by serkan-ozal — APPROVED** [review]

**Pull request merged**